### DISCUSSION TITLE
Add branding to the login screen

### DISCUSSION THREAD

**Discussion by SebastiaanKloos** [filamentphp/filament#1197] [Ideas]
I was wondering if more people need branding on the login screen.

For now I have overwritten the login partial in my project. I'd be happy to make a PR if you think this is a good idea too.

I'd love to hear what you think!

---
Create a new file `/resources/views/vendor/filament/login.blade.php` and add the following code
```php
<div class="flex items-center justify-center min-h-screen bg-gray-100 text-gray-900">
    <div class="p-2 max-w-md space-y-8 w-screen">
        <form wire:submit.prevent="authenticate" class="bg-white space-y-8 shadow border border-gray-300 rounded-2xl p-8">
            <div class="w-full flex justify-center">
                <x-filament::brand></x-filament::brand>
            </div>

            <h2 class="font-bold tracking-tight text-center text-2xl">
                {{ __('filament::login.heading') }}
            </h2>

            {{ $this->form }}

            <x-filament::button type="submit" class="w-full">
                {{ __('filament::login.buttons.submit.label') }}
            </x-filament::button>
        </form>

        <x-filament::footer />
    </div>
</div>
```

---
![CleanShot 2022-01-11 at 12 23 24](https://user-images.githubusercontent.com/6506510/148934593-f1357dd9-e28c-4194-bcdc-77108b500d3b.png)

**Comment by ralphjsmit** [comment]
I think this is a good idea Sebastiaan!

**Comment by danharrin** [comment]
This looks good for your custom logo, but how does it look with the default brand (`APP_NAME`)?

**Reply by SebastiaanKloos** [reply]
@danharrin I would suggest adding a custom partial specifically for the top of the card on the login screen like `<x-filament::brand></x-filament::brand>` is being used in the sidebar. I think that should default to a title with `APP_NAME`.

**Reply by danharrin** [reply]
What happens if you use `<x-filament::brand />` there though, does it look ok?

**Reply by SebastiaanKloos** [reply]
This one is with the default `<x-filament::brand />`. I think it's okay to re-use the <x-filament::brand />` on the login page.

<img width="840" alt="CleanShot 2022-01-11 at 19 52 09@2x" src="https://user-images.githubusercontent.com/6506510/149003779-afa91f57-d755-419c-bfbd-a55f4a45a995.png">

**Reply by danharrin** [reply]
Yeah, I don't think this looks too bad! Make a PR.
